pkg/secrets/config/config_test.go

```diff
@@ -35,11 +35,15 @@ func TestConfig(t *testing.T) {
 					})
 
 					expectedConfig := &Config{
-						PodNamespace:       testNamespace,
-						RetryCountLimit:    testRetryCountLimit,
-						RequiredK8sSecrets: testK8sSecretsList,
-						RetryIntervalSec:   testRetryIntervalSec,
-						StoreType:          storeType,
+						ContainerConfig: ContainerConfig{
+							PodNamespace:       testNamespace,
+							RetryCountLimit:    testRetryCountLimit,
+							RequiredK8sSecrets: testK8sSecretsList,
+							RetryIntervalSec:   testRetryIntervalSec,
+						},
+						SecretsConfig: SecretsConfig{
```

Comment:
do we need to add or change tests ?

Reply:
I don't believe so. The tests that exist seem to be covering the config well